linkis-dist/package/conf/version.properties

```diff
@@ -0,0 +1,2 @@
+version=BDPE-UJES_1.1.9-202306081009
```

Comment:
set to version=Linkis-x.x.x-202306081009

Reply:
done